Packwiz/pre-launch.ps1

```diff
@@ -0,0 +1,36 @@
+# By Ultrasonic1209
+
+$mods = @(
+    'ModYouDislike.jar'
+    'AnotherMod.jar'
+)
+
+# Upgrading Fabulously Optimized
+Write-Output "Checking for FO updates..."
+Set-Location ..
+
+$json = Get-Content "mmc-pack.json" | ConvertFrom-Json
+
+$table = $json.components | Where-Object { $_.cachedName -eq "Minecraft" }
+$mcVersion = $table.version
+
+if (Test-Path -Path ".\.minecraft") {
+    Set-Location ".\.minecraft"
+} else {
+    Set-Location ".\minecraft"
+}
+

```

Comment:
I don't see a checksum updater here... it will break as soon as I release new FO.

Reply:
> I don't see a checksum updater here... it will break as soon as I release new FO.

I am unable to reproduce when updating to FO 3.7.0 with the scripts.

![image](https://user-images.githubusercontent.com/44583181/166314999-0a304ee6-188a-405a-916d-2e0037e87674.png)

Reply:
> I am unable to reproduce when updating to FO 3.7.0 with the scripts.

1. Is this a new instance, meaning did you use the same instance when testing in the previous instance?
2. Did you add the checksums by yourself?

Reply:
Readding this here to not forget:
Both Windows and Linux scripts should either use or not use the checksum calculator for consistency. Whether it is needed or not should be [discussed with the packwiz dev](https://discord.gg/DcSkRF4).

Reply:
>     1. Is this a new instance, meaning did you use the same instance when testing in the previous instance?
> 
>     2. Did you add the checksums by yourself?

1. I do not understand your question.
2. No. I did not touch any packwiz-related JSON files.

Reply:
> 1. I do not understand your question.

Is this the instance you used in the version before the version you updated to now?

> 2. No. I did not touch any packwiz-related JSON files.

I meant the hashes placed inside your script

Reply:
> Is this the instance you used in the version before the version you updated to now?
Yes.
> I meant the hashes placed inside your script
There are no hashes inside either one of my scripts.

Reply:
This is very weird, but if it works properly, then it can be merged in FO I believe